File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int D;
    do
    {
        D = get_int("Height: \n");
    }
    while (D < 1 || D > 8);
  
    for (int i = 0; i < D; i++)
    {
        for (int j = 1; j < D - i; j++)
        {
            printf(" ");
        }
         
        for (int k = D - i; k <= D; k++)
        {
            printf("#");
        }
         
        printf("  ");
         
        for (int m = 0; m <= i; m++)
        {
            printf("#");
        }
         
        printf("\n");
         
    }
      
}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    print("Height:  ", end="")
    height = get_int("")
    if height > 0 and height < 9:
        break

for i in range(height):
    print(" " * (height - (i + 1)), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))